src/Provision.js

```diff
@@ -29,41 +34,108 @@ const STANDARD_DRM_CERT={
     }
   }
 };
+const OUTPUT_FILE="./src/tenant_status.json";
+
+const typeMetadata = {
+  bitcode_flags: "abrmaster",
+  bitcode_format: "builtin",
+  public: {
+    "eluv.displayApp": "default",
+    "eluv.manageApp": "default",
+  }
+};
+
+const TENANT_OPS_KEY="tenant-ops";
+const CONTENT_OPS_KEY="content-ops";
+
+const EXPECTED_SENDER_BALANCE=1; // provides 1Elv to opsKey
+// This value must be greater than 0.1 Elv
+// (checked by elv-client-js when adding as an access group member)
+const OPS_AMOUNT=0.2;
+
+// ===================================================================
+
+const isEmptyParams = (value) => {
+  return value === null || value === "" || value === false || value === "none";
+};
+
+const writeConfigToFile= (config) => {
+  fs.writeFileSync(OUTPUT_FILE, JSON.stringify(config, null, 2));
+};
+
+const checkSignerTenantAccess = async ({client, tenantId}) => {
+  let tenantAdminId = await client.userProfileClient.TenantId();
+  let tenantContractId = await client.userProfileClient.TenantContractId();
+
+  if (!tenantAdminId) {
+    throw Error("No tenant admin group set for account.");
+  }
```

Comment:
This is not required in the future - it's just vestigial  Let's not check for it

We require:
- user has tenantContractId and it is the same as the tenantId supplied in the command
- tenant has to have group 'tenant admins'
- user is a member of tenant admins of that tenant